### PR TITLE
WIP: Remove /var/lib/apt/lists/* before apt-get-update

### PR DESCRIPTION
**Pull request by adzuci**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

File: util/install/ansible-bootstrap.sh [hunk 1]
```diff
@@ -92,6 +92,7 @@ fi
 EDX_PPA="deb http://ppa.edx.org ${SHORT_DIST} main"
 
 # Upgrade the OS
+sudo rm /var/lib/apt/lists/* -vf
```
Comment (jmbowman) [inline util/install/ansible-bootstrap.sh]: Looks like it needs to be `sudo rm -r ...`: https://build.testeng.edx.org/job/build-packer-ami/14791/console

File: util/install/ansible-bootstrap.sh [hunk 2]
```diff
@@ -92,6 +92,7 @@ fi
 EDX_PPA="deb http://ppa.edx.org ${SHORT_DIST} main"
 
 # Upgrade the OS
+sudo rm -r /var/lib/apt/lists/* -vf
```
Comment (coryleeio) [inline util/install/ansible-bootstrap.sh:95]: I wouldn't sudo this, given the other lines in here you can probably assume it's already been sudoed or run as root

Comment (coryleeio) [reply util/install/ansible-bootstrap.sh:95]: Ah Jeremy suggested you sudo it, so I just made a catch 22 for you I'm worried sudo wont be installed in some contexts, and i'm worried about side effects if someone in the community runs this

Comment (coryleeio) [reply util/install/ansible-bootstrap.sh:95]: I would make this build from scratch or use a new linux ami of the correct version
It is either using an old AWS ami or using our last build AMI and eventually becoming out of date, I would recommend fixing it that way if you can then this issue shouldn't occur.

**Review by coryleeio — APPROVED** [review]
Thumbed with caveats

File: util/install/ansible-bootstrap.sh (hunk 2 above)
Comment (adzuci) [reply util/install/ansible-bootstrap.sh:95]: Working on looking at how to update the AMI.

Comment (adzuci) [reply util/install/ansible-bootstrap.sh:95]: Based on https://github.com/edx/jenkins-job-dsl/blame/master/testeng/resources/build-packer-ami.sh#L6 I think at least this packer build does build from scratch from the latest 16.04 image.

We may want to file a ticket to update build jenkins to 18.04 or 10.04 too.

**Pull request merged**